src/components/AppHighlightAlert/AppHighlightAlertWrapper.jsx

```diff
@@ -1,12 +1,16 @@
 import React, { useEffect, useState } from 'react'
 
 import { getBackupAppHighlightAlert } from 'components/AppHighlightAlert/BackupAppHighlightAlert'
+import { getGeolocationTrackingAppHighlightAlert } from 'components/AppHighlightAlert/GeolocationTrackingAppHighlightAlert'
 
 const AppHighlightAlertWrapper = ({ apps }) => {
   const [appHighlightAlerts, setAppHighlightAlerts] = useState([])
 
   useEffect(() => {
-    const appHighlightAlerts = [getBackupAppHighlightAlert()]
+    const appHighlightAlerts = [
+      getBackupAppHighlightAlert(),
+      getGeolocationTrackingAppHighlightAlert()
+    ]
```

Comment:
So the first in that array has the priority, and so on?

Reply:
Yes. We did not chose any way to set a priority for the moment.